charts/misc-apps/README.md

```diff
@@ -27,7 +27,6 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | chartmuseum.chart | string | `"chartmuseum"` | Chart |
 | chartmuseum.destination.namespace | string | `"infra-chartmuseum"` | Namespace |
 | chartmuseum.enabled | bool | `false` | Enable chartmuseum |
-| chartmuseum.repoURL | string | `"https://chartmuseum.github.io/charts"` | [repo](https://chartmuseum.github.io/charts) |
```

Comment:
seems like this broke

Reply:
same as prom teams thing ff284f8a897b501bbd81fb6753984befdf98a0e8